functions.php

```diff
@@ -49,4 +49,49 @@ function createTable(array $dbQuery) {
     return $table;
 }
 
+/**
+ * Checks length of string is less than 100 and return boolean
+ *
+ * @param string $entry string to be tested
+ * @return boolean true if length is less than 100, else false.
+ */
+function strLength(string $entry) :bool {
+    if(strlen($entry) < 100 && strlen($entry) !== 0 && isset($entry)) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+/**
+ * Checks an integer is less than 1000000000 and returns boolean
+ *
+ * @param integer $entry integer to be tested
+ * @return boolean true if integer is less than 1000000000, else returns false
+ */
+function intLength(int $entry) :bool {
+    if ($entry < 1000000000 && $entry > 0 && isset($entry)) {
```

Comment:
You should be able to safely remove the calls to isset() here - because $entry is a param, you will always need to pass something in, therefor it will always be set

Reply:
You can also simplify the code in here as per my comment on strLength